_episodes/24-practices.md

```diff
@@ -67,14 +67,8 @@ Here is a non-exhaustive list of topics:
 
 > ## Organize Your Knowledge
 >
-> Let us put the pieces together by creating a visual organization of information.  
-> We suggest doing this in two ways:
->
-> 1. If you are comfortable with / like concept maps, trying integrating all the
-> topics above into a single concept map.  
-> 2. Use the provided handout to organize topics.  Here are two examples:
->   - [Handout One](../files/handouts/Wrap-Up-doc.pdf) (with example content: [example](../files/handouts/Wrap-Up-doc-example.pdf))
->   - [Handout Two](../files/handouts/Carpentries_teaching_practices.pdf)
+> Use a concept map or other visual organiser of your choice to connect some of 
+> the concepts above. How are they related to each other? 
```

Comment:
I'm really glad to see some reframing of this final summary exercise. It can be the hardest to teach, but also most fun and rewarding. Here's how I teach this episode:

- I do "Picking up the Pieces" as follows: "Based on the content we've discussed throughout this workshop, add at least one item to each category below (we'll do this together as a class)". The two categories listed are Concepts/Theories and Tools/Practices. Everyone does a pretty good job filling in main points (takes only a few minutes), then I do a quick verbal summary offering praise for how well they've covered things.
- I give them five minutes to create a concept map. This tends to be easier since we've already put the structure of the two categories behind it. I also let them know it's ok to pick off a chunk of ideas, rather than trying to cover them all.

The feedback I've received is that they like doing an activity as a whole class. They also have a bit more scaffolding in mind, so they can more rapidly start on a concept map. Ultimately, I think it's more efficient and satisfying, and feels collaborative for the whole group. 

I'd be happy to submit a PR with suggested rewording if that would be useful.